src/main/java/Storage.java

```diff
@@ -0,0 +1,40 @@
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.reflect.TypeToken;
+
+public class Storage {
+    static Gson gson = new GsonBuilder()
+            .enableComplexMapKeySerialization()
+            .setPrettyPrinting()
+            .create();
+
+    static String userDirectory = System.getProperty("user.dir");
+
+    public static void readFromFile() {
+        try {
+            FileReader fileReader = new FileReader(userDirectory + "/data.json");
+            TaskList.initializeArray(gson.fromJson(fileReader, new TypeToken<List<Task>>() {
+            }.getType()));
+        } catch (FileNotFoundException e) {
+            TaskList.initializeArray(new ArrayList<Task>());
+        }
+    }
```

Comment:
Violates OOP principles, directly modifying a static property of another class is a big no no, and makes it very hard to unit test.